client/test/suite/client.test.ts

```diff
@@ -112,21 +111,21 @@ suite('Extension Test Suite', () => {
 */
 cts.doc(cts.uris().toArray()[12 + 19])
 `;
-        const gstate = defaultDummyGlobalState();
         const newHost: string = Math.random().toString(36);
 
-        const firstClient: ClientContext = getDbClient(queryWithDocumentsDbOverride, SJS, config, gstate);
+        const firstClient: ClientContext = getDbClient(queryWithDocumentsDbOverride, SJS, config);
         firstClient.params.host = newHost;
 
         // Verify next client is different since firstClient's params were modified
-        const secondClient: ClientContext = getDbClient(queryWithDocumentsDbOverride, SJS, config, gstate);
+        const secondClient: ClientContext = getDbClient(queryWithDocumentsDbOverride, SJS, config);
         assert.notStrictEqual(firstClient, secondClient);
         assert.notDeepStrictEqual(firstClient, secondClient);
 
+        // SKIPPING - we are not currently using a cache
```

Comment:
Can this be deleted instead of commented out? Seems irrelevant until we decide to use a cache, which may never happen.